packages/data-transport-layer/src/services/server/service.ts

```diff
@@ -106,29 +106,45 @@ export class L1TransportServer extends BaseService<L1TransportServerOptions> {
   private _initializeApp() {
     // TODO: Maybe pass this in as a parameter instead of creating it here?
     this.state.app = express()
+    if (this.options.ethNetworkName) this._initMonitoring()
+    this.state.app.use(cors())
+    this._registerAllRoutes()
+    this.state.app.use(Sentry.Handlers.errorHandler())
+  }
+
+  /**
+   * Initialize Sentry and Prometheus metrics for deployed instances
+   */
+  private _initMonitoring() {
     // Init Sentry options
+    const release = `data-transport-layer@${process.env.npm_package_version}`
```

Comment:
I think we should parse this inside of `packages/data-transport-layer/src/services/run.ts` and pass it through as part of the config so all env var/config parsing happens in a single place

Reply:
will do that!